opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -118,255 +145,122 @@ message MetricDescriptor {
-  enum Type {
-    // Do not use this default value.
+  // Interval describes how metric values are reported over time.
+  enum Interval {
+    // UNSPECIFIED is the default Interval, it MUST not be used.
     UNSPECIFIED = 0;
 
-    // Integer gauge. The value can go both up and down over time.
-    // Corresponding values are stored in Int64DataPoint.
-    GAUGE_INT64 = 1;
-
-    // Floating point gauge. The value can go both up and down over time.
-    // Corresponding values are stored in DoubleDataPoint.
-    GAUGE_DOUBLE = 2;
-
-    // Histogram gauge measurement.
-    // Used in scenarios like a snapshot of time that current items in a queue
-    // have spent there.
-    // Corresponding values are stored in HistogramDataPoint. The count and sum of the
-    // histogram can go both up and down over time. Recorded values are always >= 0.
-    GAUGE_HISTOGRAM = 3;
-
-    // Integer counter measurement. The value cannot decrease; if value is reset then
-    // start_time_unix_nano should also be reset.
-    // Corresponding values are stored in Int64DataPoint.
-    COUNTER_INT64 = 4;
-
-    // Floating point counter measurement. The value cannot decrease, if
-    // resets then the start_time_unix_nano should also be reset.
-    // Recorded values are always >= 0.
-    // Corresponding values are stored in DoubleDataPoint.
-    COUNTER_DOUBLE = 5;
-
-    // Histogram cumulative measurement.
-    // Corresponding values are stored in HistogramDataPoint. The count and sum of the
-    // histogram cannot decrease; if values are reset then start_time_unix_nano
-    // should also be reset to the new start timestamp.
-    CUMULATIVE_HISTOGRAM = 6;
-
-    // Summary value. Some frameworks implemented Histograms as a summary of observations
-    // (usually things like request durations and response sizes). While it
-    // also provides a total count of observations and a sum of all observed
-    // values, it calculates configurable percentiles over a sliding time
-    // window.
-    // Corresponding values are stored in SummaryDataPoint.
-    SUMMARY = 7;
+    // INSTANTANEOUS is a metric whose values are measured at a particular
+    // instant.
+    //
+    // If an INSTANTANEOUS metric is reported over any time interval the
+    // values are representative of distinct measurements within the
+    // interval.
+    INSTANTANEOUS = 1;
+
+    // DELTA is a metric whose values are measured and successively added
+    // over a reporting interval. The reported value represents the change
+    // from the previous interval.
+    //
+    // Each reported value for a DELTA metric over a time interval is
+    // self-contained, the values do not depend on values in other time
+    // intervals. This is unlike a CUMMULATIVE metric where values are sums
+    // of all measured events in the time interval and all previously
+    // measured values of the metric.
+    //
+    // DELTA metrics are not guaranteed to be monotonic.
+    DELTA = 2;
```

Comment:
I've never seen a system where Deltas are not monotonic, are we trying to do too much here and overgeneralize?

Reply:
delta is the sum of all observed events. If you are returning a delta value for a quantity that can both increase or decrease in the observation time this would not be monotonic.

Reply:
I think what I said is that I've never seen a system (backend: Prometheus, Stackdriver, SignalFx, etc.) that support delta for non-monotonic values. I did not say that I don't understand what you wrote, just an observation and something to think about if we need to support this if nobody needs this.

Reply:
I can speak for New Relic, in that it supports non-monotonic delta. I'm a bit surprised to hear Stackdriver no longer supports this as I have in the past sent data of this type to it.

I'm wondering if I might have been misinterpreted here though. The goal was to make sure that it is clear this type is not coupled to the monotonic refinement. This goal seems prudent with [the impending inclusion of an instrument that produces just this kind of data](https://github.com/open-telemetry/oteps/pull/98/files#diff-95b3f96b8945175022788618354f953eR25).

Reply:
I think I am fine with having this, it was mostly something that I want others to think about and see if it is useful. It seems that this is useful so we can have this :)